### PR TITLE
Implement arrow json writer

### LINKED ISSUE #492: Implement arrow json writer
Similar to CsvWriter, dispatch Polars DataFrame to arrows writer.

Source code to get started:

* [arrow json writer](https://ritchie46.github.io/polars/arrow/json/writer/index.html)
* [polars csv writer](https://ritchie46.github.io/polars/polars_io/csv/struct.CsvWriter.html)

### PR DESCRIPTION
**Pull request by marcvanheerden**
[Work in progress]

To fix #492

Feel free to comment or make suggestions on the progress so far

### CODE REVIEW

**Review by ritchie46 — COMMENTED** [review]
Nice work @marcvanheerden . I think most is already working right?

File: polars/polars-io/src/json.rs [hunk 1]
```diff
@@ -62,11 +79,39 @@
...
+use std::io::{Read, Seek, Write};
 use std::sync::Arc;
 
+// Write a DataFrame to JSON
+pub struct JSONWriter<'a, W: Write> {
+    /// File or Stream handler
+    buffer: &'a mut W,
+}
+
+impl<'a, W> SerWriter<'a, W> for JSONWriter<'a, W>
```
[the remark this answers is not in the document]
Comment (marcvanheerden) [reply polars/polars-io/src/json.rs]: thanks, I've made the change

**Comment by marcvanheerden** [comment]
I think the functionality is all there but I wasn't sure if there were any more documentation updates I need to do?

**Comment by ritchie46** [comment]
Thanks @marcvanheerden !

**Pull request merged**